specification/cosmos-db/resource-manager/Microsoft.DocumentDB/2015-04-08/cosmos-db.json

```diff
@@ -1565,6 +1942,143 @@
       },
       "description": "Represents metrics values."
     },
+    "PercentileMetricListResult": {
+      "properties": {
+        "value": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PercentileMetric"
+          },
+          "description": "The list of percentile metrics for the account."
+        }
+      },
+      "description": "The response to a list percentile metrics request."
+    },
+    "PercentileMetric": {
```

Comment:
id, name type properties missing

Reply:
any entities returned via a GET must have id, name and type top level properties.

Reply:
What if the entity doesn't return id, name or type as a top level property?